screeps-game-api/src/traits.rs

```diff
@@ -0,0 +1,72 @@
+//! Useful traits for interacting with JavaScript above what [`stdweb`] provides.
+use stdweb::unstable::TryFrom;
```

Comment:
How about `pub use` both `TryFrom` and `TryInto` from _stdweb_ here? In the event of the trait stabilizing in rust, we could have a single line to change. Moreover, it'd make it possible for people to use both without needing to use `stdweb` directly, making the API more contained.

Reply:
Sounds like a good idea!

At first glance it seemed like it would be depending too much on an "unstable" API, but we're already using it everywhere and it's already a public dependency with our implementations on public types.